README.md

```diff
@@ -1,191 +1,323 @@
+1.  [Status Page](#status-page)
+1.  [Internal Monitoring](#internal-monitoring)
+1.  [Development](#development)
+1.  [Support](#support)
+
+# Setup Guide
+
+In brief, to set up the ts-bridge app:
+
+1.  Create a GCP project that will host the app
+1.  Configure metrics for import
+1.  Deploy the app and let it auto-import your metrics every minute
+
+The following sections will guide you through this process.
+
+## Create and Set Up a Google Cloud Project
+
+We recommend making the project that hosts ts-bridge separate from the rest of
+your infrastructure so infrastructure failures will not affect monitoring and
+monitoring failues will not affect infrastructure.
+
+1.  Log in to [GCP](https://console.cloud.google.com) and
+    [create a new Google Cloud project](https://console.cloud.google.com/projectcreate)
+1.  Ensure the new project is
+    [linked to a billing account](https://console.cloud.google.com/billing/linkedaccount)
+    (Note that the Stackdriver free tier can accommodate up to about 220
+    metrics. If you have already consumed your free quota with other usage, the
+    incremental cost per metric is between US$2.32 and US$0.55 per month,
+    depending on which pricing tier you are already in.)
+1.  Enable [stackdriver monitoring](https://console.cloud.google.com/monitoring)
+    for the new project. When prompted:
+    *   Create a new stackdriver account for the project
+    *   Monitor only the new project (it should be selected by default)
+    *   Skip AWS setup and agent installation
+    *   Choose whether to receive email status reports for the project
+
+## Set Up A Dev Environment
+
+We recommend using [Cloud Shell](https://cloud.google.com/shell/) to prepare
+ts-bridge for deployment to ensure a consistent and stable working environment.
+If you need a dev environment that you can share among multiple users, consider
+using a git repository and
+[open-in-cloud-shell links](https://cloud.google.com/shell/docs/open-in-cloud-shell).
+
+1.  If you are **not** using Cloud Shell:
+    *   Install [go](https://golang.org/)
+    *   Download and install the
+        [Cloud SDK for Go](https://cloud.google.com/sdk/docs/)
+        *   Initialize with the following commands to set the linked project and
+            auth cookie:
+        *   `gcloud init`
+        *   `gcloud auth application-default login`
+    *   Download and install the
+        [original App Engine SDK for Go](https://cloud.google.com/appengine/docs/standard/go/download)
+        (Expand the "Previous App Engine SDK" header to see it)
+1.  Clone the ts-bridge source
+    *   `go get github.com/google/ts-bridge/...`
+    *   The ts-bridge source code should appear in
+        ~/gopath/src/github.com/google/ts-bridge/
+
+## End To End Test (Dev Server)
+
+1.  Ensure that you either have **Owner** permissions for the whole Cloud
+    project, or at minimum the **Monitoring Editor** role
+1.  If using a service account to authenticate, set the
```

Comment:
If this is not necessary when Cloud Shell is used, perhaps we should explicitly say so?

Reply:
which part wouldn't be necessary when using the cloud shell?

Reply:
I believe setting `GOOGLE_APPLICATION_CREDENTIALS` is not necessary while using Cloud Shell. In general, I would suggest keeping instructions simpler and only focusing on the Cloud Shell case.

Reply:
Is that true?  I thought Cloud Shell used the credentials of the logged-in user by default.  Wouldn't they need to set this variable if they're using a service account?  or are you saying that we should only focus on the case where the logged-in user is deploying with her own credentials?

Reply:
Yes, I am assuming the user would be using their personal credentials to deploy the app. What would be the use case that involves a service account and Cloud Shell?

Reply:
I was thinking of the setup for automated deployment, but you're right.  that is not needed for the initial setup that this document describes.  Removing.